packages/stockflux-core/src/services/ChildWindowLauncher.js

```diff
@@ -1,18 +1,44 @@
 import * as OpenfinApiHelpers from '../openfin-api-utils/openfinApiHelpers';
 
+const windowAlreadyExist = async (windowName, childWindows) => {
+  for (let i = 0; i < childWindows.length; i++) {
+    const childWindowOptions = await childWindows[i].getOptions();
+    if (childWindowOptions.name === windowName) {
+      return true;
+    }
+  }
+  return false;
+};
+
 const createChildWindow = async options => {
   const childWindows = await OpenfinApiHelpers.getChildWindows();
   const currentOptions = await OpenfinApiHelpers.getCurrentWindowOptions();
 
   options.uuid = currentOptions.uuid;
-  for (let i = 0; i < childWindows.length; i++) {
-    const childWindowOptions = await childWindows[i].getOptions();
-    if (childWindowOptions.name === options.name) {
-      if (childWindows[i]) {
-        childWindows[i].bringToFront();
-        return true;
+
+  if (options.allowMultiple) {
```

Comment:
Assuming there is no need to clarify `allowMultiple` on the other files because this will evaluate `undefined` as `false`, so it saves lines to just clarify the ones that are `true`?

Reply:
That was my intent, that way we do not need to change any buttons/links/window launchers we do not wish to have multiple of.

Reply:
Sounds good, just wanted to clarify